client-hc/src/test/java/com/graphhopper/api/GraphHopperGeocodingIT.java

```diff
@@ -0,0 +1,61 @@
+package com.graphhopper.api;
+
+import com.graphhopper.api.model.GHGeocodingRequest;
+import com.graphhopper.api.model.GHGeocodingResponse;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.net.SocketTimeoutException;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Robin Boldt
+ */
+public class GraphHopperGeocodingIT {
+
+    public static final String KEY = "614b8305-b4db-48c9-bf4a-40de90919939";
```

Comment:
We should avoid duplicating this info and reuse GraphHopperWebIT.KEY

Reply:
Thanks, I missed that :+1: